agent/agent.go

```diff
@@ -75,10 +75,12 @@ type Options struct {
 // Note: The agent exposes an endpoint via a TCP connection that can be used by
 // any program on the system. Review your security requirements before starting
 // the agent.
-func Listen(opts Options) (err error) {
+func Listen(opts Options) error {
 	mu.Lock()
 	defer mu.Unlock()
 
+	var err error
```

Comment:
I guess technically this could be declared before it's used on line 93 (similar to how `lc` is declared right before use).